### DISCUSSION TITLE
MIB 3.5.1 features

### DISCUSSION THREAD

**Discussion by tbilic14** [Mr-MIBonk/M.I.B._More-Incredible-Bash#195] [Q&A]
Hello,

New release say 
* Navignore java patch solution for all brands with support for CP and AA

However, when I try to apply dual navigation it says it's not supported. FW is MHI2_ER_AU57x_K3663_1_AIO.

Also, where can I read which AU57xx fixes have been done?

**Comment by harman-f** [comment]
https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/commit/ed9784594f45c77d055d7f6ca78d4ef443a3ff30
